inspirehep/base/static/less/base/variables.less

```diff
@@ -33,6 +33,11 @@
 @institutions-collection: #79B873;
 @journals-collection: #98A3A4;
 
+// Buttons colors
```

Comment:
Can you add an indication on how to use them? what color is used for what part of the button?